frontend/src/metabase/search/components/SearchResult.jsx

```diff
@@ -1,62 +1,95 @@
 import React from "react";
 import { Box, Flex } from "grid-styled";
 import styled from "styled-components";
+import { jt } from "ttag";
 
 import * as Urls from "metabase/lib/urls";
-import { color } from "metabase/lib/colors";
+import { color, lighten } from "metabase/lib/colors";
 
 import Icon from "metabase/components/Icon";
 import Link from "metabase/components/Link";
-import Tooltip from "metabase/components/Tooltip";
+import Text from "metabase/components/type/Text";
+
+import Database from "metabase/entities/databases";
+import Table from "metabase/entities/tables";
+
+function colorForType(props) {
```

Comment:
nit: `colorForType` suggests it receives `props.type`, when in fact it does more than that

Reply:
I'm not sure I fully understand the nit.

Reply:
The name `colorForType` implies it receives a `type` as the argument and makes a decision based on that. In fact it receives a whole `props` as the argument and makes a decision based on either `props.type` or `props.item.collection_position`.

(Like I said, not a big deal)